data/uhc_pack/functions/pre_generation/stage/next.mcfunction

```diff
@@ -0,0 +1,13 @@
+scoreboard players add Stage uhcPGSet 1
```

Comment:
Can `Stage uhcPGSet` be removed and make use of `Dimension uhcPGSet` instead?

Reply:
The reason for the Stage score was originally for displaying progress, or at least for debugging that the datapack was moving onto the next dimension. I think it would make configuring the generation a bit more error-prone if we were to remove this.